docs/introduction.md

```
@@ -0,0 +1,78 @@
+
+### Creating a win-to-go installation
+
+Certain laptops have one or more firmware for internal devices that must be updated using windows: it is very important you keep windows in a bootable state on a (preferably fast SSD or nvme) external disk!
+
+Use your windows installation to run rufus and create a win-to-go installation of windows.
+
+Once done start that windows installation and ensure it says it has a valid license (license should be applied from the ACPI just by booting up the installation) and install the official ASUS Armoury Crate as well as any other driver that is available via the ASUS website for your model.
+
+> [!WARNING]
+> You are supposed to use this windows installation to fully update your laptop before installing linux and regularly after!
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)